doc/elements-block-header-format.md

```diff
@@ -0,0 +1,742 @@
+| Entry Type              | Yes               | 1 byte   | `unsigned char`                 |               | `0` -> Null params, `1` -> Compact params, `2` -> Full Params                                                                                                 |
+| Signblock Script        | Entry Types 1 & 2 |          | `CScript`                       |               | "scriptPubKey" used for block signing                                                                                                                         |
+| Signblock Witness Limit | Entry Types 1 & 2 | 4 bytes  | `uint32_t`                      | little endian | Maximum size in bytes of the size of a blocksigning witness                                                                                                   |
+| Elided Root             | Entry Type 1 only | 32 bytes | `uint256`                       |               | Only non-zero when Entry Type is 1 (pruned/compact). This hash commits to the pruned fields, which are considered unchanged since the last Full params entry. |
+| Fed Peg Program         | Entry Type 2 only |          | `CScript`                       |               | The "scriptPubKey" of the fedpegscript                                                                                                                        |
+| Fed Peg Script          | Entry Type 2 only |          | `CScript`                       |               | The witnessScript for witness v0, or otherwise undefined.                                                                                                     |
+| Extension Space         | Entry Type 2 only |          | `vector<vector<unsigned char>>` |               | Interpreted as pairs of Pegout Authorization Keys (PAKs) on LiquidV1                                                                                          |
+
+---
+
+### Examples
+
+#### Pre-DynaFed
+
+| Block Height | Block Hash                                                         | Link to header hex                                                                                                                    |
+| ------------ | ------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------- |
+| `1 517 039`  | `33a0890bea61d70b9dd1bd23f12e447730336906a44cdea621cba6d65fe90013` | [blockstream.info](https://blockstream.info/liquid/api/block/33a0890bea61d70b9dd1bd23f12e447730336906a44cdea621cba6d65fe90013/header) |
+
+4 bytes - **version** (`0x20000000 | 0x02000000` - 25th bit [dynafed signalling](https://github.com/ElementsProject/elements/blob/elements-22.1/src/chainparams.cpp#L344))
+
+`00000022`
+
+32 bytes - **previous block hash**
+
+`f66be8726f4afecc9a7b087a78cd25edd53485755d26e456d546bcc0613a30f9`
```

Comment:
is it worth pointing out that this the hash here is in little-endian (reversed) order?

Reply:
that is noted on line 43

Reply:
https://github.com/ElementsProject/elements/pull/1217/files#diff-56e0e05da38ca56b8f9ab1b276cec7acb38f26eeaf3215fa88799e639a67227aR43